### ISSUE TITLE
这都一天了（2017.1.25）谷歌都上不去，但AppID已超额3个，这是怎么了？

### ISSUE THREAD

**Issue by netfun2000** [XX-net/XX-Net#4920]
XX-Net Status:

sys-platform: armv6l, Linux-4.4.44-1-ARCH-armv6l-with-glibc2.4
os-system: Linux
os-version: #1 Mon Jan 23 18:56:07 MST 2017
os-release: 4.4.44-1-ARCH
os-detail: Dist:; Version:; ID:
architecture: 32bit,
browser: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/55.0.2883.87 Safari/537.36
xxnet-version: 3.2.9

python-version: 2.7.13
openssl-version: 16.0.0 TLSv1_2 h2:alpn

ipv6-status: 0
gws-ip-num: total:3000 good:1502
network-status: OK
connected-link: new:2 used:0
worker: h1:14 h2:2
scan-ip-thread-num: 50
ip-quality: 445
is-idle: 0
block-stat: OK
proxy_state: OK
ca_state: OK
Appid_Working: true
Appids_Out_Of_Quota: true
Appids_Not_Exist: false
Using_Public_Appid: false

**Comment by jzp820927** [comment]
看状态页，没什么问题，日志呢？

**Comment by netfun2000** [comment]
XX-Net 3.2.9，一切正常，你可以访问真正的互联网了。
状态

属性	值
本地网络状态	OK
IP数量	1507
空闲	0
IP延迟	488
连接池(帮助)	新:0 h1:25 h2:6
浏览器代理设置	OK
CA证书状态(下载)	OK
扫描IP线程数(设置)	50
屏蔽状态(帮助)	OK
配置

属性	值
监听代理	0.0.0.0:8087
PAC自动代理地址	http://127.0.0.1:8086/proxy.pac
IPv6	禁用

**Comment by jzp820927** [comment]
我是说，你把日志贴出来看看

**Comment by netfun2000** [comment]
an 25 22:46:09.160 - [DEBUG] 74.125.106.237 worker close:idle timeout
Jan 25 22:46:17.168 - [DEBUG] sort ip time:384ms
Jan 25 22:46:17.219 - [DEBUG] 113.171.244.21 keep alive fail, inactive_time:154 head_timeout:100
Jan 25 22:46:17.256 - [DEBUG] report_connect_fail:113.171.244.21
Jan 25 22:46:17.277 - [DEBUG] get ip:74.125.106.237 t:398
Jan 25 22:46:18.155 - [DEBUG] create_ssl update ip:74.125.106.237 time:221 h2:0
Jan 25 22:46:22.284 - [DEBUG] 113.171.244.21 worker close:keep alive
Jan 25 22:46:25.539 - [DEBUG] 200.77.222.207 close:host pool alive_timeout
Jan 25 22:46:25.554 - [DEBUG] ssl_closed 200.77.222.207
Jan 25 22:46:28.463 - [DEBUG] sort ip time:290ms
Jan 25 22:46:28.479 - [DEBUG] get ip:58.27.61.40 t:564
Jan 25 22:46:29.339 - [INFO] scan_ip add ip:62.201.216.83 time:1697
Jan 25 22:46:29.901 - [DEBUG] sort ip time:548ms
Jan 25 22:46:29.915 - [INFO] remove_slowest_ip:110.93.194.169 handshake_time:882, fails:1
Jan 25 22:46:30.481 - [DEBUG] create_ssl update ip:58.27.61.40 time:1203 h2:1
Jan 25 22:46:32.275 - [DEBUG] ip:113.171.244.21 real fail
Jan 25 22:46:40.950 - [INFO] scan_ip add ip:1.255.22.162 time:408
Jan 25 22:46:41.757 - [DEBUG] sort ip time:801ms
Jan 25 22:46:41.798 - [INFO] remove_slowest_ip:194.182.232.90 handshake_time:1881, fails:0
Jan 25 22:46:52.964 - [INFO] scan_ip add ip:111.95.240.207 time:2635
Jan 25 22:46:53.492 - [DEBUG] sort ip time:523ms
Jan 25 22:46:53.526 - [INFO] remove_slowest_ip:111.95.240.207 handshake_time:2635, fails:0
Jan 25 22:47:10.926 - [DEBUG] sort ip time:382ms
Jan 25 22:47:10.976 - [DEBUG] get ip:1.255.22.162 t:408
Jan 25 22:47:15.013 - [DEBUG] 1.255.22.162 fail:timeout('timed out',)
Jan 25 22:47:15.019 - [DEBUG] report_connect_fail:1.255.22.162
Jan 25 22:47:15.029 - [DEBUG] get ip:200.77.222.207 t:569
Jan 25 22:47:16.277 - [DEBUG] create_ssl update ip:200.77.222.207 time:553 h2:0
Jan 25 22:47:17.328 - [DEBUG] 125.235.17.247 keep alive fail, inactive_time:153 head_timeout:100
Jan 25 22:47:17.334 - [DEBUG] report_connect_fail:125.235.17.247
Jan 25 22:47:21.924 - [DEBUG] sort ip time:634ms
Jan 25 22:47:21.940 - [DEBUG] get ip:208.65.155.82 t:601
Jan 25 22:47:22.351 - [DEBUG] 125.235.17.247 worker close:keep alive
Jan 25 22:47:24.332 - [DEBUG] create_ssl update ip:208.65.155.82 time:1993 h2:0

这些就是从“日志”页面复制出来的啊。还有其他日志？

**Comment by jzp820927** [comment]
看日志也没啥问题。你访问网站的时候，是显示什么东西？

**Comment by netfun2000** [comment]
长时间等待后，出现：

无法访问此网站

连接已重置。
请试试以下办法：
检查网络连接
检查代理服务器和防火墙
运行 Windows 网络诊断
ERR_CONNECTION_RESET

**Comment by jzp820927** [comment]
连接已重置？你浏览器没设置代理吧？只有在没走代理，直接打开网站的时候，才可能出现这种错误。

**Comment by netfun2000** [comment]
设置了，用的Proxy SwitchyOmega。

**Comment by jzp820927** [comment]
是什么网站？

**Comment by netfun2000** [comment]
google.com

**Comment by netfun2000** [comment]
![qq 20170130115500](https://cloud.githubusercontent.com/assets/4961582/22411823/13699c98-e6e3-11e6-86a6-d633b2ee52bf.png)

**Comment by netfun2000** [comment]
![1](https://cloud.githubusercontent.com/assets/4961582/22411859/56bb6440-e6e3-11e6-9ea2-ecc05c8d46bd.png)

**Comment by netfun2000** [comment]
![2](https://cloud.githubusercontent.com/assets/4961582/22412375/c85453be-e6e8-11e6-95dd-f8db8a03675c.png)

**Comment by netfun2000** [comment]
![3](https://cloud.githubusercontent.com/assets/4961582/22412432/2e3e5e2c-e6e9-11e6-9ed1-21036154ec56.png)

**Comment by jzp820927** [comment]
用下载的地址访问试试， 
https://www.google.com/ncr

**Comment by netfun2000** [comment]
依旧不行。

![5](https://cloud.githubusercontent.com/assets/4961582/22419092/cfa36b86-e716-11e6-8d8d-62e5e16cbd8d.png)

谢谢回复，祝春节快乐！

**Comment by netfun2000** [comment]
![6](https://cloud.githubusercontent.com/assets/4961582/22419174/37a6f0ea-e717-11e6-8a22-7b8651c38a7a.png)

**Comment by netfun2000** [comment]
![7](https://cloud.githubusercontent.com/assets/4961582/22419319/f179c858-e717-11e6-8e19-0dcdcb1e8ace.png)

**Comment by SeaHOH** [comment]
看起来像 IP 的问题，www.google.com.hk 走 GAE 都好好的，那就把 www.google.com 从直连中排除。

估计是升级时保留了部分设置，可能没完全合适。

**Comment by netfun2000** [comment]
![6](https://cloud.githubusercontent.com/assets/4961582/22451339/a4c6b43a-e7a8-11e6-93d2-1fcd582c3571.png)

还是不行。其实在没升级之前就这样，发现问题后才升级，结果还是这个样子。

你们的google都可以使用吗？其实，我只是用GOOGLE进行搜索，卖假药的太坑人，用不惯。

**Comment by netfun2000** [comment]
重新在Win10上部署可以使用GOOGLE，估计是升级树莓派的原因。

**Comment by jzp820927** [comment]
你的意思是，你在 win10 上可以正常用？树莓派上就不行？

**Comment by netfun2000** [comment]
是的。以前一直在树莓派上使用，做了一个GAE Server。可能是近期几天升级树莓派系统导致某些地方出现了问题。在WIN10上重新部署，没问题，正在使用。

**Comment by jzp820927** [comment]
不知道你的 树莓派装的什么系统？什么版本？

**Comment by netfun2000** [comment]
debian 已运行，重新装了树莓派系统，改用debian。系统正常。

**Comment by jzp820927** [comment]
也就是说，问题都解决了？没什么问题了？
